pages/common/ghci.md

```diff
@@ -0,0 +1,24 @@
+# ghci
+
+> The Glasgow Haskell Compiler's interactive environment.
+> More information: <https://downloads.haskell.org/ghc/latest/docs/html/users_guide/ghci.html>.
+
+- Start a REPL (interactive shell):
+
+`ghci`
+
+- Start a REPL and load the specified Haskell source file:
+
+`ghci {{source_file.hs}}`
+
+- Start a REPL and enable a language option:
+
+`ghci -X{{language_option}}`
+
+- Start a REPL and enable some level of compiler warnings to indicate potentially suspicious code:
+
+`ghci -W{{all|compat|...}}`
```

Comment:
Is `-Wall` the most used option? Otherwise you can use:
```suggestion
- Start a REPL and enable some level of compiler warnings (e.g. `all` or `compact`):

`ghci -W{{warning_level}}`
```
If you the pipe here: `{{all|compact|...}}` you should list all the available options, but there are a lot of them. Or, revert to `ghci -W{{all}}`

Reply:
I think that `-Wall` is the most used one, could be wrong though. I would still go with your suggestion if that's ok. It seems to be the overall better choice.